File: rest/taskrouter/taskchannels/instance/get/example-1/example-1.7.x.java
// Install the Java helper library from twilio.com/docs/java/install
import com.twilio.Twilio;
import com.twilio.rest.taskrouter.v1.workspace.TaskChannel;

public class GetTaskChannel {
	// Get your Account SID and Auth Token from https://twilio.com/console
	private static final String ACCOUNT_SID = System.getenv("TWILIO_ACCOUNT_SID");
	private static final String AUTH_TOKEN = System.getenv("TWILIO_AUTH_TOKEN");
	private static final String WORKSPACE_SID = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
	private static final String TASK_CHANNEL_SID = "TCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";

	public static void main(String[] args) {
		Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

		TaskChannel taskChannel = TaskChannel.updater(WORKSPACE_SID, TASK_CHANNEL_SID).fetch();

		System.out.println(taskChannel.getUniqueName());
	}

}

File: rest/taskrouter/workspaces/instance/post/example-1/example-1.7.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']
workspace_sid = 'WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'

client = Twilio::REST::Client.new(account_sid, auth_token)

workspace = client.taskrouter.v1.workspaces(workspace_sid)
                  .update(friendly_name: 'NewFriendlyName',
                          event_callback_url: 'http://requestb.in/vh9reovh')

puts workspace.friendly_name
